packages/babel-plugin-transform-flow-strip-types/src/index.js

```diff
@@ -50,7 +50,7 @@ export default function({ types: t }) {
         if (skipStrip) {
           throw path.buildCodeFrameError(
             "A @flow directive is required when using Flow annotations with " +
-              "babel-preset-react or the `requireDirective` option.",
+              "the `requireDirective` option.",
```

Comment:
Also I just found this, I think that it should be changed in this pr too. If i'm wrong please ping me and I will remove this changes.